### ISSUE TITLE
Project idea: Dictionary in python

### ISSUE THREAD

**Issue by fredcodee** [Atharv-Attri/HacktoberFest-Projects#14]
### a simple dictionary that can be accessed through the terminal 

###lists the words of a language and gives their meaning also providing information about pronunciation, origin, and usage. and also show suggestions

Checklist:
- [y ] This has not been made in the language I want to make it in
- [y ] I 100% believe this  meets quality standards

**Comment by Atharv-Attri** [comment]
Would you get their definition from a DB or from an API?

**Comment by fredcodee** [comment]
yeah i have the complied words and definitions in a database ready to be used

**Comment by Atharv-Attri** [comment]
Ok, sure!
